app/sqaws.py

```diff
@@ -165,8 +268,22 @@ def estimate_monthly_ebs_storage_price(region_name: str, instance_id: str) -> fl
     return total_gb * 0.1  # Assume EBS costs $0.1/GB/month, true as of Dec 2021 for gp2 type storage
 
 
-# Set a tag on an instance
-def set_tag(region_name: str, instance_id: str, tag_name: str, tag_value: str, dryrun: bool) -> None:
+def estimate_monthly_volume_price(volume_type: str, size: float, iops: float, throughput: float) -> float:
+    if 'gp3' in volume_type:  # gp3 type storage depends on storage, IOPS, and throughput
+        cost = size * 0.08
```

Comment:
Notably the costs are higher in some regions (like us-west-2). I think that is ok, but would be worth leaving a comment that these prices are based on us-east-1